content/docs/infrastructure-wg.md

```diff
@@ -0,0 +1,168 @@
+* build servers/machines
+* web sites
+* build system/scripts
+* code review tool
+* mailing lists
+* internet forum
+* chat
+* Documentation of the tools and their work flows.
+
+
+## Working group members
+
+The focus of IWG members will be on identifying, designing and agreeing on 
+infrastructure improvements. The implementation of the changes and the 
+operations of our infrastructure can be offloaded to community members or 
+contractors. This way we keep the workload for the working group low.
+
+Working group members should have the oppurtunity to dedicate 4-8 hours per 
+week on a regular basis in the initial phase, as we expect to have a significant 
+pile of decisions to take. It should be less than 4 hours once the roadmap has 
+become more stable.
+
+We would like to have broad and diverse community representation within the 
+working group as we are convinced such groups deliver the best results. Please
+contact us if you want to participate.
+
+Initial members:
+
+* Christian Kühnel
+* ... we're recruting plese contact us at TODO!
+
+## Proposal for getting started
+
+Proposal for steps to get the working group started.
+
+1. Advertise for the IWG and recruit initial group of >5 members by the end of 
+   2020.
+1. Agree on how we want to work and review/update this page end January 2021.
+1. Agree on infrastructure for the working group itself and set it up
+   (issue tracker, mailing list, document storage, ...) end January 2021.
+1. Agree on intial set of the most pressing issues we want to work end of 
+   February 2021.
+
+## Proposal for day-to-day work mode
+
+This section describes a proposal for the flow for the day to day work of the 
+working group. Note that only the good case is described here. A proposal can be
+cancelled in any of these steps. There might also be some iterations as some 
+parts of the SOW are not clear or feasible.
+
+1. Anyone in the LLVM community (including contractors) can propose a change by 
+   creating an issue on the issue tracker. 
+1. The working group will create an statement of work (SOW) based on the  
+   proposals they want to move forward onit. The SOW will consist of a problem 
+   description and (functional and non-functional) requirements for a solution 
+   of the problem. The SOW shall be made public and discussed with the 
+   community.
+1. Once the SOW and it’s costs are clear enough, the working group will decide 
+   if and how it shall be implemented (community contribution, paid community 
+   bounty, contractor)
+    1. If it requires spending money, the working group will propose the change 
+       to the board.
+    1. The board will then review and approve the expenditure.
+    1. The Foundation will set up the contract and manage the payments.
+1. The working group will work with the contractor/community on a day-to-day 
+   basis to get the work done.
+1. The working group will approve the work and authorize payments with the 
+   treasurer (if needed).
```

Comment:
> The numbering here seems wrong.

What do you mean? Because it's all "1."? The renderer will [automagically replace](https://guides.github.com/features/mastering-markdown/) these with the right numbering.

> there will need to be an RFP process

Yes, this is what I understood from our discussions. I added a sentence around the RFP process and that we need to work that out.